cooler_manipulation.ipynb

```diff
@@ -0,0 +1,1524 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">is this a good working directory, or should this go to ./outputs/ as suggested in this issue: https://github.com/open2c/open2c_examples/issues/24
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open2c/open2c_examples/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>